### ISSUE TITLE
app doesn't work

### ISSUE THREAD

**Issue by AkinoKawaii** [jaketr00/Discord-Bot-Client#13]
the app opened but it show the toggle developer tools, all feature doesnt work. Do you have any solutions?

**Comment by AL1L** [comment]
Have you put your token into `token.txt`?

**Comment by AkinoKawaii** [comment]
Yeah I have put the token but when I click save, it doesn't work (nothing happened). I guessed the electron app only processed the index.html, because the electron always showing the toggle developer tools.

**Comment by AL1L** [comment]
What OS are you on?

**Comment by AkinoKawaii** [comment]
Windows

**Comment by SpeedyCraftah** [comment]
OOF

**Comment by toastythetoaster** [comment]
Make sure to manually install bufferutil using `npm install bufferutil`

**Comment by iINexusIi** [comment]
i dont know if its working
